mro/json_encoder.py

```diff
@@ -0,0 +1,18 @@
+import json
+from uuid import UUID
+from datetime import datetime
+from mro.mro_dict import MroDict
+from mro.mro_list import MroList
+
+
+class JsonEncoder(json.JSONEncoder):
+
+    def default(self, o):
+        if isinstance(o, MroList):
+            return o.data
+        elif isinstance(o, MroDict):
+            return dict(o)
+        if hasattr(o, '__dict__'):
+            return o.__dict__
```

Comment:
We don't treat them differently for now, this is because we don't know the exact formats a user might use, we can look at adding it when/if we start using them.